web/templates/goal/form.html.eex

```diff
@@ -1,21 +1,30 @@
 <section class="ma3 ma4-m ma4-l center">
 
-  <p class="tl b">What do you want to achieve?</p>
-  <p class="tl">Describe a personal goal.</p>
+ <p class="tl b">What do you want to achieve?</p>
+ <p class="tl b">You may want to ensure your goal is SMART.</p>
+   <p>  
+     <strong>Specific</strong>, be very clear about what you want to achieve.<br>
+     <strong>Measurable</strong>, how will you know when you have achieved your goal? <br>
+      <strong>Achievable</strong>, avoid setting yourself a challenge that is too big. <br>
+      <strong>Resourced</strong>, do you have everything you need to achieve your goal?<br>
+      <strong>Time limited</strong>, set a reasonable time frame to reach your goal.<br> </p>
+<p class="tl">Describe a personal goal.</p>
+  
   <%= form_for @changeset, @action, fn f -> %>
       <div class=" mb3 pb3 bb b--hl-grey bw2">
         <%= textarea f, :content, placeholder: "Content", class: "db border-box hover-black w-100 ba b--black-20 pa2 br2 mb2 default-font" %>
       </div>
       <p class="b pt2 mb0">Steps to take:</p>
       <p class="mt1">List the things you could do to achieve this goal.</p>
+        <p>Come back each and tick each step off as you complete them.</p>
```

Comment:
Typo? Think it should say "Come back ~each~ and tick each step off as you complete them."

Reply:
thanks!